# request
duration prediction
---
# response
geometry_msgs/PoseStamped pose
